torch/optim/chain_lr_scheduler.py

```diff
@@ -0,0 +1,165 @@
+import math
+
+from .lr_scheduler import LambdaLR, StepLR, MultiStepLR, ExponentialLR, CosineAnnealingLR
+
+
+class LambdaChainLR(LambdaLR):
```

Comment:
This isn't really an "is-a" situation, but I guess Python doesn't really give us any way to actually do private inheritance lol.